mlir/include/mlir/IR/AffineExprVisitor.h

```diff
@@ -418,6 +418,14 @@ class SimpleAffineExprFlattener
                                              AffineExpr localExpr);
 
 private:
+  /// Flatten binary expression `expr` and add it to `result`. If `expr` is a
+  /// dimension, symbol or constant, we add it to appropriate index in `result`.
+  /// Otherwise we add it in the local variable section. `lhs` and `rhs` are the
```

Comment:
It's unclear here what operands of `expr` are! An AffineExpr doesn't have any operands. Rephrase.